### PR TITLE
Fix naming conventions in 'Refine per-vertex input variable'

### PR DESCRIPTION
**Pull request by trenouf**
Also slightly simplify the implementation code.

See the original change in #1555.

### CODE REVIEW

**Comment by trenouf** [comment]
@AMD-dwang Can you confirm please that I have not broken the functionality in your local testing? Thanks.

**Comment by amdvlk-admin** [comment]
### Test summary for commit 9a95abef0a5f9113a9c9ee837e53526ca7b3bbe9
<details><summary>CTS tests (Failed: 0/179976)</summary><ul><li>Built with version 1.3.0.0</li>
<details open><summary>Rhel 8.2, Gfx10</summary><ul>

<li>Passed: 35765/63748 (56.1%)</li>
<li>Failed: 0/63748 (0.0%)</li>
<li>Not Supported: 27983/63748 (43.9%)</li><li>Warnings: 0/63748 (0.0%)</li>
</ul></details>

<details open><summary>Ubuntu 18.04, Gfx9</summary><ul>

<li>Passed: 27857/52478 (53.1%)</li>
<li>Failed: 0/52478 (0.0%)</li>
<li>Not Supported: 24621/52478 (46.9%)</li><li>Warnings: 0/52478 (0.0%)</li>
</ul></details>

<details open><summary>Ubuntu 20.04, Gfx8</summary><ul>

<li>Passed: 38010/63750 (59.6%)</li>
<li>Failed: 0/63750 (0.0%)</li>
<li>Not Supported: 25740/63750 (40.4%)</li><li>Warnings: 0/63750 (0.0%)</li>
</ul></details>


</ul></details>

**Comment by amdvlk-admin** [comment]
### Test summary for commit 7c3e5c895c9be87527da05d428168bc9791aec3e
<details><summary>CTS tests (Failed: 0/179976)</summary><ul><li>Built with version 1.3.0.0</li>
<details open><summary>Rhel 8.2, Gfx10</summary><ul>

<li>Passed: 35765/63748 (56.1%)</li>
<li>Failed: 0/63748 (0.0%)</li>
<li>Not Supported: 27983/63748 (43.9%)</li><li>Warnings: 0/63748 (0.0%)</li>
</ul></details>

<details open><summary>Ubuntu 18.04, Gfx9</summary><ul>

<li>Passed: 27857/52478 (53.1%)</li>
<li>Failed: 0/52478 (0.0%)</li>
<li>Not Supported: 24621/52478 (46.9%)</li><li>Warnings: 0/52478 (0.0%)</li>
</ul></details>

<details open><summary>Ubuntu 20.04, Gfx8</summary><ul>

<li>Passed: 38010/63750 (59.6%)</li>
<li>Failed: 0/63750 (0.0%)</li>
<li>Not Supported: 25740/63750 (40.4%)</li><li>Warnings: 0/63750 (0.0%)</li>
</ul></details>


</ul></details>

File: lgc/builder/InOutBuilder.cpp [hunk 1]
```diff
@@ -251,19 +114,52 @@ Value *InOutBuilder::CreateReadPerVertexInput(Type *resultTy, unsigned location,
...
+    // These are the non-parity modes that just read a single input.
     result = readInput(vertexIndex);
+    break;
+
+  default:
+    // The remaining modes are the parity modes that read two inputs and select between them.
+    // Fix the order.
+    switch (primType) {
+    case PrimitiveType::TriangleFan:
+      oddOffset = provokingVertexMode == ProvokingVertexLast ? 1 : 2;
```
Comment (AMD-dwang) [inline lgc/builder/InOutBuilder.cpp:140]: evenOffset is missing.
`evenOffset = provokingVertexMode == ProvokingVertexLast ? 2 : 0;`

**Comment by amdvlk-admin** [comment]
### Test summary for commit 57b4d089cdfe585808cbc9cbdb08b293a30bfb92
<details><summary>CTS tests (Failed: 0/179976)</summary><ul><li>Built with version 1.3.0.0</li>
<details open><summary>Rhel 8.2, Gfx10</summary><ul>

<li>Passed: 35765/63748 (56.1%)</li>
<li>Failed: 0/63748 (0.0%)</li>
<li>Not Supported: 27983/63748 (43.9%)</li><li>Warnings: 0/63748 (0.0%)</li>
</ul></details>

<details open><summary>Ubuntu 18.04, Gfx9</summary><ul>

<li>Passed: 27857/52478 (53.1%)</li>
<li>Failed: 0/52478 (0.0%)</li>
<li>Not Supported: 24621/52478 (46.9%)</li><li>Warnings: 0/52478 (0.0%)</li>
</ul></details>

<details open><summary>Ubuntu 20.04, Gfx8</summary><ul>

<li>Passed: 38010/63750 (59.6%)</li>
<li>Failed: 0/63750 (0.0%)</li>
<li>Not Supported: 25740/63750 (40.4%)</li><li>Warnings: 0/63750 (0.0%)</li>
</ul></details>


</ul></details>

**Review by AMD-dwang — APPROVED** [review]
The code looks clearer. Thanks.

**Review by amdrexu — APPROVED** [review]

**Pull request merged**